core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java

```diff
@@ -657,6 +658,7 @@ public final Sql expr(String expr) {
     sql(sql).ok();
   }
 
+  @Disabled
```

Comment:
Nit. Lets stay consistent and add annotations after the comments.

Reply:
Done